frontend/app/controllers/spree/checkout_controller.rb

```diff
@@ -76,106 +76,118 @@ def ensure_valid_state
       end
     end
 
-      # Should be overriden if you have areas of your checkout that don't match
-      # up to a step within checkout_steps, such as a registration step
-      def skip_state_validation?
-        false
-      end
+    # Should be overriden if you have areas of your checkout that don't match
+    # up to a step within checkout_steps, such as a registration step
+    def skip_state_validation?
+      false
+    end
 
-      def load_order_with_lock
-        @order = current_order(lock: true)
-        redirect_to spree.cart_path and return unless @order
-      end
+    def load_order_with_lock
+      @order = current_order(lock: true)
+      redirect_to(spree.cart_path) && return unless @order
+    end
 
-      def ensure_valid_state_lock_version
-        if params[:order] && params[:order][:state_lock_version]
-          @order.with_lock do
-            unless @order.state_lock_version == params[:order].delete(:state_lock_version).to_i
-              flash[:error] = Spree.t(:order_already_updated)
-              redirect_to checkout_state_path(@order.state) and return
-            end
-            @order.increment!(:state_lock_version)
+    def ensure_valid_state_lock_version
```

Comment:
Assignment Branch Condition size for ensure_valid_state_lock_version is too high. [19.42/15]